src/neptune/internal/backends/neptune_backend.py

```diff
@@ -305,6 +305,8 @@ def search_leaderboard_entries(
         query: Optional[NQLQuery] = None,
         columns: Optional[List[str]] = None,
         limit: Optional[int] = None,
+        sort_by: Optional[str] = None,
+        sort_by_column_type: Optional[AttributeType] = None,
```

Comment:
From my point of view, type of sorted column is only a part of internal implementation of specific API so this should be a detail of `HostedNeptuneBackend` instead of being a part of whole interface.